.github/workflows/create-release.yml

```
@@ -0,0 +1,85 @@
+          VERSION=$(jq -r '.version // empty' package.json)
+          if [ -z "$VERSION" ] || [ "$VERSION" = "null" ]; then
+            echo "::error::version field not found or empty in package.json"
+            exit 1
+          fi
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: Check release state
+        id: state
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:10:34: Double quote to prevent globbing and word splitting

(shellcheck)